tcms/testruns/tests/test_permissions.py

```diff
@@ -56,3 +58,39 @@ def verify_post_with_permission(self):
         self.assertRedirects(
             response,
             reverse('testruns-get', args=[self.test_run.pk]))
+
+
+class TestRunTestCase(tests.PermissionsTestCase):
+    permission_label = 'testruns.add_testrun'
+    url = reverse('testruns-new')
+    http_method_names = ['post']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_run = factories.TestRunFactory(summary='Old summary')
+        cls.url = reverse('testruns-edit', args=[cls.test_run.pk])
```

Comment:
Why override the `url` attribute? 

`cls.url` here and the `url` on line 65 is exactly the same thing.